l1-l2-deposit-withdrawal/scripts/example.js

```diff
@@ -63,16 +63,21 @@ async function main() {
     'L1 ERC20', //name
   )
   await L1_ERC20.deployTransaction.wait()
+  console.log(`   @addr ${L1_ERC20.address}`)
 
   // Deploy the paired ERC20 token to L2.
   console.log('Deploying L2 ERC20...')
   const L2_ERC20 = await factory__L2_ERC20.connect(l2Wallet).deploy(
     '0x4200000000000000000000000000000000000010',
+    // The first value is to check the case of an invalid L1. The second value
```

Comment:
I don't understand this 3 line-comment ?

Reply:
Sorry, clarified.